js/image.js

```diff
@@ -0,0 +1,19 @@
+(function ($) {
+	var retries = 5;
+	$( document).ready(function(){
+		$('img').bind('error', function() {
+			var image = $(this);
+			var imgSrc = image.attr('src');
+			if (retries > 0 && typeof imgSrc !== undefined) {
```

Comment:
`typeof` is always a string so `typeof imgSrc !== undefined` is always true

I think you meant `typeof imgSrc !== 'undefined'` here, but you could as well safely use `imgSrc !== undefined` because `imgSrc` is declared in the line above.

Even better is using something like `is.set()`, in case you have such a uitlity libary included in your project.